py-polars/polars/functions/aggregation/vertical.py

```diff
@@ -40,7 +40,7 @@ def all(
 
     Otherwise, this function computes the bitwise AND horizontally across multiple
     columns.
-    **This functionality is deprecated**.
+    **This functionality is deprecated**, use ``pl.all_horizontal`` instead.
```

Comment:
Can we specify that only `pl.all()` for horizontal operations is deprecated, `pl.all() as alias for `pl.col("*")` is not.

Reply:
I'd say that is pretty clear if you read the whole docstring?

Reply:
I think my IDE thinks differently. :sweat_smile: Somehow it regards the whole function as deprecated.

Reply:
Hm I'll add some further clarification, for the confused PyCharm users among us :)

Reply:
![image](https://github.com/pola-rs/polars/assets/3023000/5bbe2ab0-3674-4a30-a09f-88b5a580641d)

@stinodego this is what I mean

Reply:
Yeah I understood that - but do you know PyCharms logic for determining what's "deprecated"? Because clearly, it's flawed. We might be able to work around it though, if we figure out their logic.

Reply:
Yes, rephrasing it somehow?

Reply:
I think PyCharm just detects that the function raises a DeprecationWarning and then marks it as deprecated. Not taking into account code paths. I don't think it has anything to do with the docstring.

Reply:
In that case we could add an indirection. E.g. put the deprecation warning logic in another dummy function and call that function.

Reply:
I'll look into this and open a separate PR if I find a solution.